packages/core/src/account.ts

```diff
@@ -27,6 +27,18 @@ export class AccountSecrets extends Serializable {
     favorites = new Set<VaultItemID>();
 }
 
+/** Various application settings */
+export class AccountSettings extends Serializable {
+    /** Enable checking for weak passwords */
+    securityReportWeak = true;
+    /** Enable checking for reused passwords */
+    securityReportReused = true;
+    /** Enable checking for compromised passwords */
+    securityReportCompromised = true;
+    /** Enable emails for failed login attempts */
+    failedLoginAttemptNotifications = true;
+}
+
```

Comment:
Just a thought, but do you think it might make sense to group these into subproperties/classes? Something like

```ts
export class SecurityReportSettings extends Serializable {
    weakPasswords = true;
    reusedPasswords = true;
    compromisedPaswords = true;
}

export class NotificationSettings extends Serializable {
    failedLoginAttempts = true;
}

export class AccountSettings extends Serializable {
    @AsSerializable(SecurityReportSettings)
    securityReport = new SecurityReportSettings();

    @AsSerializable(NotificationSettings)
    notifications = new NotificationSettings();
}
```
Yes it's a bit more boilerplate but would keep things a bit tidier which would be nice especially considering that there will probably a bunch more settings in the future.